docs/templates/model-serving/deployment_api.md

```diff
@@ -0,0 +1,9 @@
+# Deployment
+
```

Comment:
Missing the creation section. We should mention both model.deploy() and ms.create_deployment(). For example in the hopsworks-api docs for an execution, I mention you create an Execution out of a Job object. https://robzor92.github.io/hopsworks-api/2.6.0-SNAPSHOT/generated/api/executions/#run